change/@fluentui-eslint-plugin-f001f787-94f9-4cd3-979c-2cf86eb276d1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: Update ESLint config to include story and test files ",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "behowell@microsoft.com",
+  "dependentChangeType": "none"
```

Comment:
Note: I manually changed `dependentChangeType` to `none` since this doesn't affect code in packages that use eslint-plugin.